build/bin/network-validator.go

```diff
@@ -86,11 +91,35 @@ func TestEndpoints(config reachabilityConfig) {
 }
 
 func ValidateReachability(host string, port int) error {
+	var err error
 	endpoint := fmt.Sprintf("%s:%d", host, port)
+	httpClient := http.Client{
+		Timeout: *timeout,
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
We likely do not want to do this. If we disable TLS verification for all endpoints, it could (and likely will) still result in the cluster failing to provision. The only endpoints that need TLS inspection disabled are the four domains here https://docs.openshift.com/rosa/networking/configuring-cluster-wide-proxy.html#cluster-wide-proxy-network-prereqs_cluster-wide-proxy-configuration. The remainder should absolutely verify that they can connect using trusted certificates or the trust bundle provided by the end user.

Reply:
:+1: fixing